openbas-front/src/public/components/channels/ChannelNewspaper.js

```diff
@@ -42,6 +40,18 @@ const ChannelNewspaper = ({ channelReader }) => {
   const [userId] = useQueryParameter(['user']);
   const [currentArticle, setCurrentArticle] = useState(null);
   const isDark = theme.palette.mode === 'dark';
+  const channelColor = (type) => {
+    switch (type) {
+      case 'newspaper':
+        return '#3f51b5';
+      case 'microblogging':
+        return '#00bcd4';
+      case 'tv':
+        return '#ff9800';
+      default:
+        return '#ef41e1';
+    }
+  };
```

Comment:
You can create an utiliy method to avoid duplication

Reply:
It's done